src/section-exponent-rules-and-scientific-notation.ptx

```diff
@@ -37,200 +37,40 @@
+        <p>In the section <xref ref="section-introduction-to-exponent-rules">Section</xref>, we introduced three basic rules involving products and exponents.  We'll begin with a brief recap and explanation of these three exponent rules for multiplication.</p>
+
+        <list xml:id="list-exponent-rules-recap">
+            <caption>Summary of Exponent Rules</caption>
+                <dl>
+                    <li xml:id="item-exponent-rules-product-rule">
+                        <title>Product Rule</title>
+                        <p>When you're multiplying two expressions that have the same base, you can simplify the product by adding the exponents. <me>
+                        x^m \cdot x^n = x^{m+n}</me></p>
+                    </li>
+                    <li xml:id="item-exponent-rules-power-to-a-power-rule">
+                        <title>Power to a Power Rule</title>
+                        <p>When you have a base raised to an exponent and that expression is raised to another exponent, you multiply the exponents. <me>\left(x^m\right)^n = x^{m \cdot n}</me></p>
+                    </li>
+                    <li xml:id="item-exponent-rules-product-to-a-power-rule">
+                        <title>Product to a Power Rule</title>
+                        <p>When you have a product raised to an exponent, you apply the exponent to each factor in the product. <me>\left(x\cdot y\right)^n = x^{n}\cdot y^{n}</me></p>
+                    </li>
+                </dl>
+        </list> 
 
             <!-- TODO: note, Ross didn't eliminate all of the instances where this problem would occur yet. Sorry.
 
              Figure out what we want to do with this type of thing. The problems aren't written to the exactitude that it makes it clear how the answers should be displayed. And problems where 200^3 is the answer could easily be made less terrible. -->
                 <!-- <p>At this point we have to note that in some expressions we have numbers raised to large enough powers that actually writing down the number in standard form is difficult or unnecessary. For example, if you have <m>7^6</m> in your answer, we will usually leave it in that form rather than calculating <m>117649</m>. We leave it up to your discretion as to exactly when a number is <q>too large</q> to calculate but a general rule might be that if the number is smaller than <m>200</m> then you should definitely evaluate it, and anything larger than <m>1000</m> should be left in exponent form. Values in between <m>200</m> and and <m>1000</m> are left to your judgment or other requirements.</p> -->
-        </paragraphs>
 
 
-        <exercise>
+<!--         <exercise>
             <webwork source="BasicAlgebra/Exponents/exponentsMultiplication25.pg" />
         </exercise>
-
+ -->
+ <!--TODO: ask Alex how to display steps 1 and 3 correctly-->
```

Comment:
Is it possible to describe here what is not being displayed quickly? Just wondering if I should pull and compile to see.

Reply:
OK, change things like `$step1 = Compute("$var**($a*$b)*$var**($c*$d)");` to just make a string for LaTeX. Using double quotes means the perl variables will be replaced with their values:`$step1 = "$var^{$a \cdot $b} \cdot $var^{$c \cdot $d}";`.